lib/features/guide/widgets/about_us/models/link_data.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:collection/collection.dart';
+
+import '../../../../../config/ui_config.dart';
+
+class LinkData {
+  String iconUrl;
+  String? url;
+
+  static const _icons = {
+    "facebook": "assets/icons/ic_fb.png",
+    "instagram": "assets/icons/ic_insta.png",
+    "linkedin": "assets/icons/ic_linkedin.png",
+    "github": "assets/icons/ic_github.png",
+    "mailto:": "assets/icons/ic_email.png",
+    "pwr.edu.pl": "assets/icons/ic_web.png",
```

Comment:
it this `pwr.edu.pl` necessary? I believe when nothing is found, we have a defaultIconUrl which is basically this web icon